apps/sap-commerce-cloud/frontend/README.md

```diff
@@ -6,15 +6,11 @@ In the project directory, you can run:
 
 ### Install packages
 
-`npm i` 
-
-### Set development ENV
-
-Copy the contents of .env.development.sample into a new file called .env.development (calls to SAP will only work from localhost with this added)
```

Comment:
I verified that the app runs just fine without this, so I also went ahead and removed the `env.development.sample` file from the app in this PR.